src/index.ts

```diff
@@ -0,0 +1,37 @@
+import http = require("http");
+import fs = require("fs");
+
+const postData: http.RequestListener = (
+  request: http.IncomingMessage,
+  response: http.ServerResponse
+) => {
+  if (request.method === "POST") {
+    let body = "";
+    request.on("data", (data) => {
+      body += data;
+    });
+    request.on("end", () => {
+      const preparedData = dataToUpperCase(body) + "\n";
+      fs.appendFileSync("..\\data\\data.txt", preparedData);
```

Comment:
Very strange path.
Why do you need double slash?

Reply:
Changed
(I peeked it up at the Node.js Path docs at Windows vs. POSIX section. Can`t figure out why I had an issue with a path like this, just a few hours before, immediately after porting to Windows from Ubuntu.)